ISSUE_TEMPLATE/generic_issue.yml

```diff
@@ -0,0 +1,56 @@
+name: Generic Container Image Issue
+description: File a generic issue here
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report this issue to us. We appreciate the feedback and use reports such as this one to guide our efforts at improving our products. That being said, this bug tracking system is not a mechanism for requesting support, and we are not able to guarantee the timeliness or suitability of a resolution.
+
+        If this issue is critical or in any way time sensitive, please raise a ticket through the regular Red Hat support channels to ensure it receives the proper attention and prioritization to assure a timely resolution.
+
+        For information on how to contact the Red Hat production support team, please visit:
+            https://access.redhat.com/support
+
+  - type: dropdown
+    attributes:
+      label: Container platform
+      description: Is your issue related to running containers on OCP or by podman/docker?
+      multiple: true
+      options:
+        - OCP 3
+        - OCP 4
+        - Podman/Docker
+
+  - type: textarea
+    attributes:
+      label: Version
+      description: What versions are affected?
+
+  - type: dropdown
+    attributes:
+      label: Compose
```

Comment:
I wouldn't call it compose, "compose" is an internal thing that is not visible outside. "OS version" might be more suitable here. It's till not entirely clear to me though -- do we ask about OS version of the host or the container? It might be also made clear.

Reply:
fixed by [42308ac](https://github.com/sclorg/.github/pull/9/commits/42308acd501d95e6aaaf5b3368cc6a261eb8f0c1).